package.json

```diff
@@ -53,6 +53,7 @@
     "tiny-invariant": "^1.3.1"
   },
   "devDependencies": {
+    "@aws-sdk/client-route-53": "^3.259.0",
```

Comment:
This is a dependency, not a dev dependency

Reply:
oh, this is a mistake. I probably didn't notice that I was adding `-D` mindlessly. Will fix this.